components/service/glean/src/main/java/mozilla/components/service/glean/CommonMetricData.kt

```diff
@@ -35,6 +35,8 @@ interface CommonMetricData {
     val name: String
     val sendInPings: List<String>
 
+    val identifier: String get() = if (category.isEmpty()) { name } else { "$category.$name" }
```

Comment:
nit: do you really need to wrap the branches with `{}`?

Reply:
The defeat/ktlint gods have decreed it to be so

Reply:
I liked your earlier spelling of "defeat" better.  That's how it makes me feel ;)

Reply:
The irony of auto-correct while githubbing on my phone...  perhaps Sigmund Freud would have something to say about that.